index.html

```diff
@@ -43,7 +43,210 @@ <h1 class="company-logo">2Win!</h1>
 <body>
   <h2>Shares</h2>
   <section class="card-section">
-    <article class="card-container">
+    <article class="card-container one">
+      <div class="image-container">
+        <img class="card-image" src="assets/berkeley.jpeg" alt="pic of berkeley">
+      </div>
```

Comment:
I know in the accessibility lesson they mentioned divs not being as recognizable to the screen reader?  Maybe change one of two of these within your card to sections?